bingocard4astropymeeting.html

```diff
@@ -52,7 +52,8 @@
   "We will wait a few more mins",
   "AFK when called on",
   "Blockchain",
-  "Robotic audio"];
+  "Robotic audio",
+  "Reread own forgotten issue/PR"];
```

Comment:
```suggestion
  "(Re)read own forgotten issue/PR"];
```
could have been just "Read" – @eteq might be able to open PRs without reading; but it's perhaps better to allow both not to make it too hard.